google-cloud-spanner/src/main/resources/com/google/cloud/spanner/connection/ClientSideStatements.json

```diff
@@ -361,6 +369,23 @@
 	    "allowedValues": "'(([a-zA-Z][a-zA-Z0-9_\\-]{1,63})|())'",
 	    "converterName": "ClientSideStatementValueConverters$StringValueConverter"
 	  }
+	},
+	{
+		"name": "SET RPC_PRIORITY = 'HIGH'|'MEDIUM'|'LOW'",
```

Comment:
As you are adding a new statement to the client side statements set, you also need to run the following command in the root of the google-cloud-spanner project:

`mvn -P generate-test-sql-scripts compile`

See also https://github.com/googleapis/java-spanner/blob/b6db9b1524760f4fd158bbf5b2689211d2ed55e7/google-cloud-spanner/src/test/java/com/google/cloud/spanner/connection/ClientSideStatementsTest.java#L39

This will generate test statements for the new statement(s) in the `ClientSideStatementsTest.sql` file. I had a quick look locally, and:
1. It seems that we forgot to do this also for the `statistics_package` statement that was added recently. That means that you will be getting some additional unrelated changes in the test script.
2. The generated tests will fail, as the current implementation does not allow the priority value to be specified in lower case (or anything other than all upper case).